src/assertions/redteam.ts

```diff
@@ -21,11 +21,11 @@ export const handleRedteam = async ({
   ) {
     const storedResult = providerResponse.metadata.storedGraderResult;
     return {
+      ...storedResult,
```

Comment:
This re-ordering fixes a bug where assertions were undefined. See the commit message – https://github.com/promptfoo/promptfoo/pull/5562/commits/ac037ecb990f8ef7c74aeaac1cdfb722e2c56ee8 – for more detail.